tests/CommandTest.php

```diff
@@ -113,4 +115,32 @@ public function testMakeOnly()
         $this->assertFileDoesNotExist('tests/fixtures/NovaPostTest/create_post_response.json');
         $this->assertFileDoesNotExist('tests/fixtures/NovaPostTest/update_post_request.json');
     }
+
+    public function testCallWithNotDefaultConfig()
+    {
+        $rootUrl = vfsStream::setup('root', null, [
+            'config' => [
+                'entity-generator.php' => "<?php return ['test' => 'original'];",
+            ],
+            'routes' => [
+                'api.php' => "",
+            ],
+        ])->url();
+
+        $this->app->instance('path.base', $rootUrl);
```

Comment:
I already see something like this in the parent setup method https://github.com/RonasIT/laravel-entity-generator/blob/58553cf7c73155dcc392367addfd93686bebcaf0/tests/TestCase.php#L29

do we need to make this here or we can use already existed attributes?